File: Basic Coding Problems/02 If - Elses/034 Write a program to check whether the triangle is equilateral, isosceles or scalene triangle/Java/triangle_type.java
import java.util.Scanner;

public class triangle_type {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the three sides of the triangle: ");
        int sideA = scanner.nextInt();
        int sideB = scanner.nextInt();
        int sideC = scanner.nextInt();

        if (sideA == sideB && sideB == sideC) {
            System.out.println("The triangle is equilateral.");
        } else if (sideA == sideB || sideA == sideC || sideB == sideC) {
            System.out.println("The triangle is isosceles.");
        } else {
            System.out.println("The triangle is scalene.");
        }
    }
}

File: Basic Coding Problems/02 If - Elses/034 Write a program to check whether the triangle is equilateral, isosceles or scalene triangle/C++/triangle_type.cpp
#include <iostream>

using namespace std;

int main() {
    int side_a, side_b, side_c;

    cout << "Enter the three sides of the triangle: ";
    cin >> side_a >> side_b >> side_c;

    if (side_a == side_b && side_b == side_c) {
        cout << "The triangle is equilateral." << endl;
    } else if (side_a == side_b || side_a == side_c || side_b == side_c) {
        cout << "The triangle is isosceles." << endl;
    } else {
        cout << "The triangle is scalene." << endl;
    }

    return 0;
}